test/db/cmd/dwarf

```diff
@@ -256,8 +256,8 @@ EXPECT=<<EOF
 |           0x000052b4      488b442450     mov rax, qword [n]
 |           0x000052b9      48c744246001.  mov qword [var_60h], 1
 |           0x000052c2      4889442468     mov qword [var_68h], rax
-|           0x000052c7      488b7c2460     mov rdi, qword [var_60h]
-|           0x000052cc      488b742468     mov rsi, qword [var_68h]
+|           0x000052c7      488b7c2460     mov rdi, qword [var_60h]    ; int64_t arg1
```

Comment:
woot! why is this working now?

Reply:
As stated in issue description sdb_querys used previosly was vulnerable to query injection. Seems likely that something broke due to that especially with rust names which can have a bunch of special symbols.